include/SimpleAmqpClient/Channel.h

```diff
@@ -108,6 +108,7 @@ class SIMPLEAMQPCLIENT_EXPORT Channel : boost::noncopyable {
     std::string vhost;  ///< Virtualhost on the broker. Default '/', required.
     int port;           ///< Port to connect to, default is 5672.
     int frame_max;      ///< Max frame size in bytes. Default 128KB.
+    bool is_publisher_confirms = false; ///< Whether or not the channel is created in publisher confirms mode.
```

Comment:
I guess now you can default it to true ? As in its a bit different to set a bool flag in code to false than it is to disabling behavior by adding a flag